safe/definitions/hazard_classifications.py

```diff
@@ -4,7 +4,7 @@
 See https://github.com/inasafe/inasafe/issues/2920#issuecomment-229874044
 to have a table showing you classes of each kind of hazard.
 """
-from safe.definitions import concepts, small_number
```

Comment:
If you can copy/paste also here with comments in this file, it's quicker, for developers.